bt: add BT_CLR_ALL behaviour (#2037)
zmkfirmware/zmk: docs/docs/behaviors/bluetooth.md

computer/laptop/keyboard should receive the keyboard input; many of the commands here operate on those profiles.

:::note Connection Management
When pairing to a host device ZMK saves bond information to the selected
profile. It will not replace this when you initiate pairing with another device.

> To pair with a new device select an unused profile with `BT_SEL`, `BT_NXT` or
> `BT_PRV` bindings, or by clearing an existing profile using `BT_CLR` or `BT_CLR_ALL`.

Reviewer: ```suggestion
To pair with a new device, select a profile that doesn't have a pairing with `BT_SEL`, `BT_NXT` or
`BT_PRV` bindings, or clear an already paired profile using `BT_CLR` or `BT_CLR_ALL`.
```
Existing issues but would be good to correct while touching here.
